chain/actions/src/node.rs

```diff
@@ -117,8 +117,6 @@ mod tests {
 
     #[async_std::test]
     async fn test_announce() {
-        let _ = env_logger::builder().is_test(true).try_init();
```

Comment:
Because it does not work with tracing the way you'd need for tests. For debugging, you could import the `tracing-test` library and then wrap any test you want to observe in a detail with the `#[traced_test]` macro. `env_logger` is not the most ideal tool to see the detail of test you want.

Ref https://docs.rs/tracing-test/0.2.4/tracing_test/